sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -100,4 +100,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <profiles>
```

Comment:
Don't all HTTP based clients need to be updated with this profile before this change is merged? Otherwise won't there be failing live tests.